compiler+runtime/src/cpp/jank/read/lex.cpp

```diff
@@ -574,7 +574,9 @@ namespace jank::read::lex
           {
             return ch.expect_err();
           }
-          else if(std::iswspace(static_cast<int>(ch.expect_ok().character)))
+          // NOTE: Does not handle unicode space characters, though Clojure does
```

Comment:
1. Please only use `/* */` style comments. Also, comments are documentation, so please always use complete sentences, correct capitalization, and necessary punctuation. Imagine your comments are snippets from a book explaining why the code is as it is.

2. We have this `iswspace` function, which supports Unicode. We're using it here, but failing on anything which isn't an ASCII space. Why aren't we just accepting anything which is `iswspace`?

3. If we do, please add tests for those Unicode whitespace cases. If we don't, the comment this needs to be a TODO, since lack of parity with Clojure means we're not done yet.